examples/quickstarts/README.md

```diff
@@ -18,6 +18,7 @@ instantiating them.
 * [Django](https://raw.githubusercontent.com/openshift/django-ex/master/openshift/templates/django-postgresql.json) - Provides a basic Django (Python) application with a PostgreSQL database. For more information see the [source repository](https://github.com/openshift/django-ex).
 * [Django persistent](https://raw.githubusercontent.com/openshift/django-ex/master/openshift/templates/django-postgresql-persistent.json) - Provides a basic Django (Python) application with a persistent PostgreSQL database. Note: requires available persistent volumes.  For more information see the [source repository](https://github.com/openshift/django-ex).
 * [Httpd](https://raw.githubusercontent.com/openshift/httpd-ex/master/openshift/templates/httpd.json) - Provides a basic Httpd static content application. For more information see the [source repository](https://github.com/openshift/httpd-ex).
+* [Nginx](https://raw.githubusercontent.com/openshift/nginx-ex/master/openshift/templates/nginx.json) - Provides a basic Nginx static content application. For more information see the [source repository](https://github.com/openshift/nginx-ex).
```

Comment:
not sure where you created the nginx example, but this openshift/nginx-ex doesn't exist.

Reply:
https://github.com/sclorg/nginx-ex/

Reply:
I do not have access to create repos in `openshift`.

Reply:
@bparees, will you copy `sclorg/nginx-ex` to `openshift/nginx-ex` or should I just change the README.md?
All other example apps are in `openshift/...`